client/lib/types.ts

```diff
@@ -23,7 +23,7 @@ export interface Bootstrapper<T> {
   update (cache: DBCache<T>): Promise<void>
 }
 
-export interface DBCache<T extends { [key: string]: any }>{
+export interface DBCache<T extends Record<string, any>>{
```

Comment:
Update `any` to `unknown`

Reply:
Unfortunately this would cause a chain reaction of all the places `DBCache` is used becoming broken. It's best to keep it out of the scope of current refactor.